scripts/verify-status.sh

```diff
@@ -3,42 +3,56 @@
 echo "Checking status of POST Jobs for Eventing-Manager"
 
 REF_NAME="${1:-"main"}"
-TIMEOUT_TIME="${2:-120}"
+TIMEOUT_TIME="${2:-240}"
 INTERVAL_TIME="${3:-3}"
 
 # Generate job Status URL
 STATUS_URL="https://api.github.com/repos/kyma-project/eventing-manager/commits/${REF_NAME}/status"
 
+# Dates
+START_TIME=$(date +%s)
+TODAY_DATE=$(date '+%Y-%m-%d')
+
+# Retry function
 function retry {
-	local start_time=$(date +%s)
+
 	# Get status result
 	local statusresult=$(curl -L -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" ${STATUS_URL})
 	# Get overall state
-	local fullstatus=$(echo $statusresult | jq '.state' | tr -d '"')
-
-	while [ "$fullstatus" == "pending" ]; do
-		current_time=$(date +%s)
-		elapsed_time=$((current_time - start_time))
+	fullstatus=$(echo $statusresult | jq '.state' | tr -d '"')
+	local tudayprowrunstate=$(echo $statusresult | jq --arg t $TODAY_DATE '.statuses[]|select(.created_at | contains($t))' | jq '.state' | tr -d '"')
+	echo $statusresult | jq --arg t $TODAY_DATE '.statuses[]|select(.created_at | contains($t))' | jq '[.]'
```

Comment:
Question: why do we not use snake_case for theses variables too?

Reply:
Fix it